frontend/pages/org.js

```diff
@@ -0,0 +1,93 @@
+import renderPage from '../renderPage.js'
+import { getDefaultClubImageURL, doApiRequest } from '../utils'
+import React, { Component } from 'react'
+import s from 'styled-components'
+import Header from '../components/ClubPage/Header.js'
+import InfoBox from '../components/ClubPage/InfoBox.js'
+import SocialIcons from '../components/ClubPage/SocialIcons.js'
+import clubEx from './club_tree_example.json'
+import OrgChildren from '../components/OrgPage/OrgChildren.js'
+import OrgTabs from '../components/OrgPage/OrgsTabs.js'
+import { EMPTY_DESCRIPTION } from '../utils'
+
+class Org extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      club: null,
+      children: null,
+    }
+  }
+
+  componentDidMount() {
+    doApiRequest(`/clubs/${this.props.query.club}/?format=json`)
+      .then(resp => {
+        return resp.json()
+      })
+      .then(data => {
+        this.setState({ club: data })
+      })
+    doApiRequest(`/clubs/${this.props.query.club}/children/?format=json`)
+      .then(resp => {
+        return resp.json()
+      })
+      .then(data => {
+        this.setState({ children: data.children })
+      })
+  }
+  render() {
+    const { club, children } = this.state
+    if (!club) {
+      return (
+        <div className="has-text-centered">
+          <h1 className="title is-h1">Loading...</h1>
+        </div>
+      )
+    }
+    if (!children) {
+      return (
+        <div className="has-text-centered">
+          <h1 className="title is-h1">Waiting for some great content...</h1>
+          <p>Thank you for your patience!</p>
+        </div>
+      )
+    }
+    return (
+      <div style={{ padding: '30px 50px' }}>
+        <Header
+          club={club}
+          userInfo={this.props.userInfo}
+          favorites={this.props.favorites}
+          updateFavorites={this.props.updateFavorites}
+        />
+        <div></div>
+        <div className="columns">
+          <div className="column is-4">
```

Comment:
Seems like there might be some duplicate code here. @ezwang do you think it's worth refactoring this type of stuff into a component used across several pages or no?